drivers/filesystems/fastfat/finfo.c

```diff
@@ -66,6 +66,12 @@ const char* FileInformationClassNames[] =
     "FileMaximumInformation"
 };
 
+BOOLEAN IsDots(UNICODE_STRING File)
```

Comment:
I think I'd prefer if this were named even more explicitly (such that nobody reading the calling code has to look up what this function does), e.g. `IsDotOrDotDot`

Reply:
Great idea. Done.